components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/util/AccountUtil.java

```diff
@@ -47,4 +52,16 @@ public static String getTenantDomain(UserStoreManager userStoreManager) {
         }
     }
 
+    public static String getConnectorConfig(String key, String tenantDomain) throws IdentityEventException {
+        try {
+            Property[] connectorConfigs;
+            IdentityGovernanceService identityGovernanceService = AccountServiceDataHolder.getInstance()
+                    .getIdentityGovernanceService();
+            connectorConfigs = identityGovernanceService.getConfiguration(new String[]{key,}, tenantDomain);
```

Comment:
Fixed in https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/33/